vembrane/globals_whitelist.py

```diff
@@ -0,0 +1,108 @@
+import math
+import re
+
+# The builtins list below was generated with:
+#    python -c 'print(
+#        *sorted(o for o in dir(__builtins__) if o.islower() and not o.startswith("_")),
+#        sep="\n",
+#    )'
+_builtins = {
+    obj.__name__: obj
+    for obj in (
+        # abs,
+        all,
+        any,
+        # # ascii,
+        # bin,
+        # bool,
+        # # breakpoint,
+        # # bytearray,
+        # # bytes,
+        # # callable,
+        # chr,
+        # # classmethod,
+        # # compile,
+        # # complex,
+        # # copyright,
+        # # credits,
+        # # delattr,
+        dict,
+        # # dir,
+        # divmod,
+        # enumerate,
+        # # eval,
+        # # exec,
+        # # exit,
+        # filter,
+        float,
+        # format,
+        # frozenset,
+        # # getattr,
+        # # globals,
+        # # hasattr,
+        # # hash,
+        # # help,
+        # hex,
+        # # id,
+        # # input,
+        int,
+        # isinstance,
+        # # issubclass,
+        # iter,
+        # len,
+        # # license,
+        list,
+        # # locals,
+        map,
+        max,
+        # # memoryview,
+        min,
+        # next,
+        # # object,
+        # oct,
+        # # open,
+        # ord,
+        # pow,
+        # # print,
+        # # property,
+        # # quit,
+        # range,
+        # # repr,
+        # reversed,
+        # round,
```

Comment:
```suggestion
        round,
```
(if there's no round in math, didn't check)

Reply:
Just checked, there isn't.